### ISSUE TITLE
[$80][MSFT-95] [Challenges Feed][Chrome] 'Non-text content' and 'Info and Relationship' and 'Link purpose(In context)' and 'language of the page' and  'label and instructions', 'Name, Role, Value' not match as per WCAG 2.0 A guidline

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2801]
**Steps to Reproduce**
1) Got to the page "https://www.topcoder.com/challenges" in edge browser.
2) Use to check accessibility compliance website 'http://www.cynthiasays.com/' and enter menederoty things with webpage URL e.g. Email ID, Web Page URL(https://www.topcoder.com/challenges),
Compliance mode(WCAG 2.0A and WCAG 2.0 AA)
3) Tick the term and condition 
4) Submit the Test you site button and observe the scan output.

**Expected Result**
1) Non-text content - expected to control through on label and title 
2) Info and relationation- expected to control through on label and title
3) Link Purpose- expected to anchor have alternative  text.
4) Language of the page- Must be specify LANG and XML:LANG  attribute
5) Labels and  Instructions- expected to control through on label and title
6) Name role and value- expected to control through on label and title
http://www.cynthiasays.com/report.aspx?id=10642

**Actual Result**
Found six issue related to the Non-text content, Info and relationships and Link purpose, language of the page, Labels and Instructions, Name, role, value as per

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: Web Accessibility Toolbar (WAT)

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=8ab28edc%2Dd11f%2D4bb3%2Db9cf%2Da743b86a634f&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fkmurti%2FAccessibility%2F3

**Comment by ThomasKranitsas** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096115 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096115 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096115 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096897 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096897 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096897 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@sushilshinde @veshu @codeMinter unable to test the website using the provided website http://www.cynthiasays.com. No results are shown for the scan. It is very likely that all these errors stated above are fixed in other PRs. How shall I proceed?

**Comment by crazyk07** [comment]
Duplicate

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
